### DISCUSSION TITLE
How do we save multiple images and then load them?

### DISCUSSION THREAD

**Discussion by anirudhlath** [yasirkula/UnityNativeCamera#95] [Q&A]
I tried using the example code, but it replaces the pictures in path. How can we click a picture, save it to a specific location with a specific name and then access that location in another script to show a scroll view of all the pictures that were taken?

**Comment by yasirkula** [comment]
You're right, NativeCamera overwrites the previous image. You need to use File.Copy or File.Move to copy that file to your desired location before taking another image. For example, you can save them to a folder inside persistentDataPath. Then you can load them from there via NativeCamera.LoadImageAtPath.

**Reply by anirudhlath** [reply]
Perfect, I have been trying all sort of things such getrawtexturedata to save a byte array and all that, but had issue with read/write there. I guess File.Copy will work just fine! I will update what I have tomorrow and see how it goes! I am also trying to save the path in PlayerPrefs. I am new to unity programming and it's quite extensive with no enough documentation 😅

**Reply by yasirkula** [reply]
I believe this'll be a good learning experience for you then ^^

**Reply by anirudhlath** [reply]
Absolutely. Saving worked, now loading and showing it 😮‍💨
